build.sh

```diff
@@ -8,16 +8,16 @@
 # built, which is in the current directory where this script resides. [The
 # requirement is to define this variable in the U-Boot directory is a Siklu
 # design flaw that should be fixed.]
-export PROJECT_ROOT_DIR="$(pwd)"
+#export PROJECT_ROOT_DIR="$(pwd)"
```

Comment:
why comment out it? Do we need PROJECT_ROOT_DIR to compile u-boot?

Reply:
This script is for building U-Boot outside of the regular portfolio build system.
When you do this, the script has no idea where there is a portfolio clone to use for the version script that it needs to complete the link. So you have to tell the script where to look manually and there's now way to guess.